### ISSUE TITLE
Build warning (Policy CMP0042 is not set: MACOSX_RPATH)

### ISSUE THREAD

**Issue by sashafrey** [bigartm/bigartm#453]
```
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   artm

This warning is for project developers.  Use -Wno-dev to suppress it.
```

**Comment by sashafrey** [comment]
Run `cmake ..` on MAC OS X to reproduce  this issue.

**Comment by JeanPaulShapo** [comment]
Fixed in #528
